app/helpers/decidim/forms/application_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Forms
+    # Custom helpers, scoped to the forms engine.
+    module ApplicationHelper
+      # Show cell for selected models
+      def show_represent_user_group?
+        model_name = questionnaire_for.model_name.element
+
+        permited_models.include?(model_name)
+      end
+      alias show_public_participation? show_represent_user_group?
+
+      def permited_models
+        %(meeting)
+      end
+
+      def invalid?(responses)
+        bool = false
```

Comment:
```suggestion
                responses.map { |res| res.errors.any? }.any?
```

You can return an array of bool and just check if one of them is true